src/components/Modal/a11yDialog.jsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import A11yDialogLib from "a11y-dialog";
+import PropTypes from "prop-types";
+
+const useA11yDialogInstance = () => {
+  const [instance, setInstance] = React.useState(null);
+  const container = React.useCallback(node => {
+    if (node !== null) setInstance(new A11yDialogLib(node));
+  }, []);
+
+  return [instance, container];
+};
+
+export const useA11yDialog = props => {
```

Comment:
Sorry for the pettiness - is it a component or a hook? if it is a hook we usually do no use propTypes and we use  this structure for attributes ^^" 
```
export const useA11yDialog = {attribute1, attribute2} => {
```
But it is super cool! And we can also use it in other dialogs when we will do a refactor

Reply:
👍